src/main/java/com/pub/wysio/service/CustomerService.java

```diff
@@ -0,0 +1,51 @@
+package com.pub.wysio.service;
+
+import com.pub.wysio.entity.Customer;
+import com.pub.wysio.repository.CustomerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CustomerService {
+    @Autowired
+    private CustomerRepository repository;
```

Comment:
for the same reason I commented in the controller, this should be `customerRepository`